renovate-base.json5

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:recommended"
```

Comment:
Why this?

Reply:
This is a new pipeline that I have added and that I have added here: https://github.com/newrelic/coreint-automation/pull/33/files#diff-efda70da6ee709cf755707663c052d5f163fa082bb1e6c441fb08cd36d3341f5R24

Renovate bot changed that and the bot complained that "base" does not exist anymore and that it is being migrated to "recommended".

I changed it to have a cleaner CI run.